server/models/index.js

```diff
@@ -0,0 +1,399 @@
+import sequelize, { Op } from '../lib/sequelize';
```

Comment:
:no_entry_sign: It looks like this file is the result of a merge conflict, only `server/models/index.ts` should be there